Dockerfile

```diff
@@ -13,9 +14,10 @@ FROM --platform=$BUILDPLATFORM tonistiigi/xx:${XX_VERSION} AS xx
 # osxcross contains the MacOSX cross toolchain for xx
 FROM crazymax/osxcross:${OSXCROSS_VERSION} AS osxcross
 
-FROM --platform=$BUILDPLATFORM golang:${GO_VERSION}-alpine AS gobase
+FROM --platform=$BUILDPLATFORM golang:${GO_VERSION}-bullseye AS gobase
```

Comment:
Do we know if `-buster` would work on both "bullseye" and "buster", or would that break? I recall the containerd maintainers had to downgrade their machines to ubuntu 18.04 to make them more compatible; https://github.com/containerd/containerd/compare/v1.6.7...v1.6.8

Reply:
Yes I was also wondering what was our best option. I will try and let you know.

Reply:
`ppc64le`, `s390x` and `armel` (arm/v6) cross pkgs are not available in buster :persevere:

Reply:
Alright; if we already used bullseye, then it's probably fine